docs: Quality Inspection on Job Card (#201)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/manufacturing/job-card.md

> <img class="screenshot" alt="Create Shareholder" src="/docs/assets/img/manufacturing/work-order-job-card-completion.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Work Order Job Card Completion" src="/docs/assets/img/manufacturing/work-order-job-card-completion.png">
```

## 3. Features

### 3.1 Tracking Quality Inspection

> Introduced in Version 13

> For production orders, the quality of in-process(semi-finished) goods also needs to be tracked. It is defined by the process(operation) done on it which is in turn defined in the Job Card. In-process tests are different than incoming and outgoing material tests. Monitoring quality during manufacturing helps to make sure that the finished product produced is of the desired quality. You can create Quality Inspection for the Production Item against Job Card.

Reviewer: ```suggestion
For production orders, the quality of in-process (semi-finished) goods also needs to be tracked. It is defined by the process (operation) performed on it which is in turn defined in the Job Card. In-process tests are different than incoming and outgoing material tests. Monitoring quality during manufacturing helps to make sure that the finished product produced is of the desired quality. You can create a Quality Inspection for the Production Item against the Job Card.
```

> <img class="screenshot" alt="Create Shareholder" src="/docs/assets/img/manufacturing/qi-against-job-card.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Quality Inspection Against Job Card" src="/docs/assets/img/manufacturing/qi-against-job-card.png">
```

> <img class="screenshot" alt="Create Shareholder" src="/docs/assets/img/manufacturing/qi-link-in-job-card.png">

Reviewer: ```suggestion
<img class="screenshot" alt="Quality Inspection link in Job Card" src="/docs/assets/img/manufacturing/qi-link-in-job-card.png">
```
